src/Platform/Microsoft.Testing.Platform/Hosts/ServerTestHost.cs

```diff
@@ -93,13 +96,36 @@ public ServerTestHost(
 
     protected override bool RunTestApplicationLifeCycleCallbacks => true;
 
+    public string Uid => nameof(ServerTestHost);
+
+    public string Version => AppVersion.DefaultSemVer;
+
+    public string DisplayName => PlatformResources.ServerTestHostDisplayName;
+
+    public string Description => PlatformResources.ServerTestHostDescription;
+
     private void OnCurrentDomainUnhandledException(object sender, UnhandledExceptionEventArgs e)
-        => _logger.LogWarning($"[ServerTestHost.OnCurrentDomainUnhandledException] {e.ExceptionObject}{_environment.NewLine}IsTerminating: {e.IsTerminating}");
+    {
+        _logger.LogWarning($"[ServerTestHost.OnCurrentDomainUnhandledException] {e.ExceptionObject}{_environment.NewLine}IsTerminating: {e.IsTerminating}");
+
+        // Looks like nothing in this message to really be localized?
+        // All are class names, method names, property names, and placeholders. So none is localizable?
+        ServiceProvider.GetOutputDevice().DisplayAsync(
+            this,
+            new WarningMessageOutputDeviceData(
+                $"[ServerTestHost.OnCurrentDomainUnhandledException] {e.ExceptionObject}{_environment.NewLine}IsTerminating: {e.IsTerminating}"))
+            .GetAwaiter().GetResult();
```

Comment:
@MarcoRossignoli is this better to do this or use await on async void? I think I recall the await async void was the "recommended" way for eventing methods in WPF but I might say something wrong.

Reply:
@Evangelink I initially had it `async void` as I'm coming from the background that in general async void is okay for events in UI frameworks (e.g, WPF). Marco suggested using `GetAwaiter().GetResult()` instead.

Reply:
Ok so let's go with this suggestion. Thanks!